node-operators/block-producers/onboarding/account-setup.md

```diff
@@ -0,0 +1,152 @@
+
+## Important Variables {: #important-variables }
+
+When establishing eligibility to produce blocks, there are a couple of variables to be aware of:
+
+- **Minimum self-delegation** - there is a minimum amount you must self-delegate to be considered eligible
+- **Number of active candidates** - the top candidates by stake that can actively produce blocks
+
+|          Variable           |                          Value                           |
+|:---------------------------:|:--------------------------------------------------------:|
+|   Minimum self-delegation   | {{ networks.dancebox.staking.min_self_del.dance }} DANCE |
+| Number of active candidates | {{ networks.dancebox.staking.top_eligible_candidates }}  |
+
+## Map an Account to Your Block Producer Node {: #map-account }
+
+The first step is a two-step process that generates [session keys](https://wiki.polkadot.network/docs/learn-keys#session-keys){target=\_blank} and maps the session keys to your account. Session keys are used to perform network operations, such as signing blocks. By mapping the session key to your account, you create an association between your account and your block-producing node.
+
+You will need to create session keys for your primary and backup servers. Each of your servers, your primary and backup, should have its own unique keys. Since the keys never leave your servers, you can consider them a unique ID for that server.
+
+### Generate Session Keys {: #generate-session-keys }
+
+Before generating session keys, you must be [running a block-producing node](/node-operators/block-producers/onboarding/run-a-block-producer){target=\_blank}.
+
+To generate session keys, you'll send an RPC call, using the `author_rotateKeys` method, to your node's HTTP endpoint. For reference, if your collator's HTTP endpoint is at port `9944`, the JSON-RPC call might look like this:
+
+```bash
+curl http://127.0.0.1:9944 -H \
+"Content-Type:application/json;charset=utf-8" -d \
+  '{
+    "jsonrpc":"2.0",
+    "id":1,
+    "method":"author_rotateKeys",
+    "params": []
+  }'
+```
+
+Your hex-encoded session keys will be printed to the terminal in the `"result"` field.
+
+--8<-- 'code/node-operators/block-producers/onboarding/account-setup/terminal/generate-session-keys.md'
+
+Make sure you write down your session keys; you'll need to map your session keys to your account in the next section.
+
+### Map Session Keys {: #map-session-keys }
+
+To perform the next step and map your session keys to your account, head to [Polkadot.js Apps](https://polkadot.js.org/apps/?rpc=wss://fraa-dancebox-rpc.a.dancebox.tanssi.network#/extrinsics){target=\_blank}, click on the **Developer** tab, select **Extrinsics** from the dropdown, and take the following steps:
+
+1. Select your account, which should be the same account that you previously self-delegated
+2. Select the **session** module and the **setKeys** extrinsic
+3. For **keys**, enter your session keys
```

Comment:
The UI label is plural, but it's just one key. wdyt? 
```suggestion
3. For **keys**, enter your session key
```

Reply:
I think we should just leave it as keys. That's how we have it on Moonbeam and is what Polkadot does too 🤷‍♀️